qcweb/templates/layout.html

```diff
@@ -27,9 +27,9 @@
           </button>
           <div class="collapse navbar-collapse" id="navbarToggle">
             <div class="navbar-nav mr-auto">
-              <a class="nav-item nav-link" href="/">Home</a>
-              <a class="nav-item nav-link" href="/query">Query</a>
-              <a class="nav-item nav-link" href="/table">Table</a>
+              <a class="nav-item nav-link" href="{{ url_for('home') }}">Home</a>
+              <a class="nav-item nav-link" href="{{ url_for('query') }}">Query</a>
+              <a class="nav-item nav-link" href="{{ url_for('table') }}">Table</a>
               <a class="nav-item nav-link" href="/plot">Plot</a>
```

Comment:
Why didn't you use `url_for` for `/plot`?

Reply:
If changed to `url_for` for plot, it will show alt graph instead of src image.